test/e2e/resources/pipeline_execution.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: PipelineExecution
+metadata:
+  name: $PIPELINE_EXECUTION_RESOURCE_NAME
+spec:
+  pipelineExecutionDisplayName: $PIPELINE_EXECUTION_RESOURCE_NAME
+  pipelineName: serial-inference-pipeline
```

Comment:
Is this created in the other test?

Reply:
pipeline creation missing and use a smaller pipeline?